dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1752,19 +1745,19 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 	}
 
 	if (chatCommand == "rollloot" && entity->GetGMLevel() >= eGameMasterLevel::OPERATOR && args.size() >= 3) {
-		uint32_t lootMatrixIndex = 0;
-		uint32_t targetLot = 0;
-		uint32_t loops = 1;
+		const auto lootMatrixIndex = GeneralUtils::TryParse<uint32_t>(args[0]);
+		const auto targetLot = GeneralUtils::TryParse<uint32_t>(args[1]);
+		const auto loops = GeneralUtils::TryParse<uint32_t>(args[2]);
```

Comment:
These should be unrolled so we arent constantly constructing and destructing optional objects in the loop below

Reply:
I hate that I'm basically replying to all these with "I'm not sure what you mean," but I'm not sure what you mean here.

Reply:
This one is fine, pretty sure I saw a copy in the loop below at some point or something idk, i dont see a copy happening in the loop at a second glance

Reply:
I'll double check for you

Reply:
Think I found a potential copy in a for-ranged loop. Not related to my changes, but I'll fix it just in case.